mono/utils/mono-error.c

```diff
@@ -593,7 +593,7 @@ mono_error_set_not_verifiable (MonoError *oerror, MonoMethod *method, const char
 static MonoString*
 string_new_cleanup (MonoDomain *domain, const char *text)
 {
-	MonoError ignored_err;
+	ERROR_DECL_NAME (ignored_err)
```

Comment:
Personal preference: I like to write the trailing semicolons myself.   (For the selfish reason that it plays nicer with the auto-indentation in C mode in Emacs)

Reply:
I'm ok either way. One rule is "is it is a wierd syntactic construct, then it is a wierd syntactic construct", that is, you know, what is ERROR_DECL_NAME (foo); supposed to parse as? It looks like a function call, with a variable, but it isn't a function call, and the variable doesn't yet exist. So it is just a wierd text producing macro, it can and does look like anything. But if w want to pretend it is an expression and fool emacs and turn it into a statement, that can be ok.